### ISSUE TITLE
Add term tight craniocervical junction

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10437]
**Preferred term label:**
Tight craniocervical junction

**Synonyms**
craniocervical junction constriction, tight craniovertebral junction

**Definition (free text, please give PubMed ID)**
Congenital or acquired abnormalities of the occipital bone, foramen magnum, or first two cervical vertebrae that decrease the space for the lower brain stem and cervical cord. These malformations can lead to cranial nerve compression, vertebral artery compression, and obstructive hydrocephalus. PMID1566672

**Parent term (use hpo.jax.org/app)**
Abnormal cervical spine morphology HP:0046508

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achondroplasia (OMIM100800) PMID33103849

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000920
- label: Craniocervical junction constriction
- parents: [Abnormal cervical spine morphology (HP:0046508)](https://hpo.jax.org/app/browse/term/HP:0046508)
- pmid: PMID:1566672
- def: Congenital or acquired abnormalities of the occipital bone, foramen magnum, or first two cervical vertebrae that decrease the space for the lower brain stem and cervical cord. These malformations can lead to cranial nerve compression, vertebral artery compression, and obstructive hydrocephalus.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10437
- synonyms: Tight craniovertebral junction(NONE)|Tight craniocervical junction(NONE)
